interceptors/ratelimit/ratelimit.go

```diff
@@ -9,17 +9,17 @@ import (
 )
 
 // Limiter defines the interface to perform request rate limiting.
-// If Limit function return true, the request will be rejected.
+// If Limit function return an error, the request will be rejected.
```

Comment:
💪🏽

Reply:
```suggestion
// If Limit function returns an error, the request will be rejected with XXX status code and the provided error as the response.
```
We need to find solid XXX that makes sense I believe.

Reply:
I think I didn't saw that LOL. 
:grin: